sdk/include/ndk/peb_teb.h

```diff
@@ -248,9 +302,15 @@ typedef struct STRUCT(_TEB)
     PTR(PVOID)             glTable;
     PTR(PVOID)             glCurrentRC;
     PTR(PVOID)             glContext;
-    NTSTATUS               LastStatusValue;
+    ULONG                  LastStatusValue;
```

Comment:
The symbols tend to be bad at this kind of thing... do you know if this is semantically an NTSTATUS? If so, leaving it as NTSTATUS here could be better.

Reply:
It might be semantically an NTSTATUS, but technically it is an unsigned long, while NTSTATUS is a long.